### PR TITLE
The Syndicate Uplink Expansion Project

### PR DESCRIPTION
**Pull request by hyperjll**
## About The Pull Request

Currently bugfixing and merge conflict... fixing? Sorry for the lack of a description, looking to fix conflicts and bugs at the moment before describing everything. Just chill out will ya?

## Changelog

:cl:
add: Adds a LOT of new syndicate uplink item listings.
/:cl:

### CODE REVIEW

**Comment by YesterdaysPromise** [comment]
the description doesn't say much at all

**Comment by Jacquerel** [comment]
My man I don't know if something went wrong with your branch but this touches 288 files and you didn't write any information about it in the PR body, are you sure you were ready to post this?

**Comment by Sadboysuss** [comment]
can you list all the items it adds please and explain it in detail?

File: code/datums/diseases/advance/symptoms/syndicate.dm
Comment (YesterdaysPromise) [inline code/datums/diseases/advance/symptoms/syndicate.dm:1]: Don't like these descriptions. Such a field for creativy, such a waste. Could have at least name-dropped Interdyne.

**Comment by hyperjll** [comment]
Will update the description later, hoping to fix things FIRST before describing everything (+making the changelog).

**Comment by Jacquerel** [comment]
> Will update the description later, hoping to fix things FIRST before describing everything (+making the changelog). And yes, i did mean to post this, it's a DRAFT.

How many items are you adding here?

**Comment by YesterdaysPromise** [comment]
At least 30 it looks like Jacq. With several virus mutation tools and genetic mutations bulking it up.

**Comment by LemonInTheDark** [comment]
This is not being reviewed. Way too much shit from somone I don't know adding stuff for a reason I don't understand

**Comment by Jacquerel** [comment]
For your own benefit: This is going to have to be multiple pull requests.
Probably a _lot_ of pull requests.

Nobody is going to want to review a PR which add 17,000 lines of code across 300 files.
This is impossible to do with any kind of rigour.

At the same time, for _your_ benefit, it means that a PR adding 30 items won't be blocked on account of the fact that there is _one_ thing in there that we don't like.
Combining a whole lot of things that don't rely on the same code into the same PR isn't a sustainable practice from either end.

**Comment by hyperjll** [comment]
Fine. I'll go the extra mile and try separating everything.

**Comment by YesterdaysPromise** [comment]
Way over 30 things it looks like and some of it is... quite weird? Like straight up slime extracts, seeds of the weaponized plants, spare null rod, the fucken DIE OF FATE... It feels like it has everything

**Comment by LemonInTheDark** [comment]
image
Even if you separate it the chances of it being merged is VERY low.

Like, we don't have a pressing need for new uplink content and you haven't justified why we'd want this. I don't just want new syndicate stuff

**Comment by YesterdaysPromise** [comment]
I can *somewhat* understand most things, but who hit you with the Chaplain's multitool nullrod in the head to get the DIE OF FATE, the item in there for just 25TC?

**Comment by breadcrombs** [comment]
holy moly, new traits, organs, guns, calibers, melee and more

**Comment by hyperjll** [comment]
Working on atomizing, pray for my sanity.

**Comment by Sakamoto4ka** [comment]
yoooo im gonna check dis

**Comment by breadcrombs** [comment]
> image Even if you separate it the chances of it being merged is VERY low.
> 
> Like, we don't have a pressing need for new uplink content and you haven't justified why we'd want this. I don't just want new syndicate stuff for its own sake
> 
> Also where is this from? there's so much did you make it all yourself?

i think this is inspired from goon, theres a botany chainsaw and also a ak caliber definition

**Comment by CCC23** [comment]
HOLY FUCK MAN THIS SHIT LOOKS GOOD AS FUCK split some of this up cause DAYUM all ya gotta do is split it and make a pdf explaining shit and that looks NICE

**Pull request closed without merging**